contracts/bridge/contracts/common/message/OutboundLane.sol

```diff
@@ -245,6 +248,21 @@ contract OutboundLane is IOutboundLane, MessageVerifier, TargetChain, SourceChai
         }
     }
 
+    function on_messages_delivered(DeliveredMessages memory confirmed_messages) internal {
+        for (uint64 nonce = confirmed_messages.begin; nonce <= confirmed_messages.end; nonce ++) {
+            uint256 offset = nonce - confirmed_messages.begin;
+            bool dispatch_result = ((confirmed_messages.dispatch_results >> offset) & 1) > 0;
+            address submitter = messages[nonce].payload.sourceAccount;
```

Comment:
is OutboundLane or other outbound lanes possible to be message submitter? 

Is submitter a contract? request more document for submitter

Reply:
1. outbound lanes have no interface to summit the message, so they can not.
2. The submitter could be a contract or just an EOA address.